<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
	xmlns:context="http://www.springframework.org/schema/context"
	xmlns:util="http://www.springframework.org/schema/util"
	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
		">
	
	<context:component-scan base-package="org.is.lnu.edbo.handler"/>
	<import resource="edbo-exception-handlers.xml"/>
	<bean id="edboAuthentificationService" class="org.is.lnu.edbo.service.login.DefaultAuthentificationService">
		<property name="serviceManager" ref="edboGuidesServiceManager" />
		<property name="applicationKey" value="${edbo.application.key}" />
		<property name="clearPreviousSession" value="${edbo.clear.previous.session}"/>
	</bean>

	<bean id="edboPersonService" class="org.is.lnu.edbo.service.person.DefaultEdboPersonService">
		<property name="serviceManager" ref="edboPersonServiceManager" />
		<property name="applicationKey" value="${edbo.application.key}" />
		<property name="clearPreviousSession" value="${edbo.clear.previous.session}"/>
		<property name="exceptionHandler" ref="edboPersonExceptionHandler"/>
	</bean>
 	
 	<bean id="edboLanguageService" class="org.is.lnu.edbo.service.language.DefaultEdboLanguageService"/>
 	<bean id="edboActualDateService" class="org.is.lnu.edbo.service.date.DefaultActualDateService"/>	

</beans>
